### ISSUE TITLE
git lfs required in jenkins base install

### ISSUE THREAD

**Issue by samuelbernardolip** [indigo-dc/jenkins-pipeline-library#37]
We should review if git lfs should be optional and add another attribute for that in config.yml or assume it as a requirement.
In current code, when checkout runs it also fetches the lfs part from the repository if exists.
This requires that Jenkins nodes have git-lfs support (installed).

**Comment by davrodgon** [comment]
Yes, this should be either documented as a requirement, or made optional (and document that option). The latter is probably preferable given that not all repositories actually require this. 
In the current situatiion, a Jenkins installation without git-lfs installed in the node thirows errors that are not easy to debug like:
hudson.plugins.git.GitException: Could not checkout  <some revision hexadecimal ID>
With no reference to LFS in the error message.
This is more confusing given that previous git operations have been successful.

**Comment by samuelbernardolip** [comment]
Miss to review this for release 2.1.0.
This is important for next release, so setting label as critical.
